assets/js/components/ViewSelector.js

```diff
@@ -38,6 +37,9 @@ module.exports = class ViewSelector {
       return;
     }
 
+    // matches top padding in scss
+    let topSpaceWhenFixed = 30;
```

Comment:
Should be `const` if the value doesn't change

Reply:
As above these have been brought in from the update from when we last updated last year. But yes agree. It should be const